src/uniprotkb/components/statistics/HistoricalReleasesEntriesLinePlot.tsx

```diff
@@ -0,0 +1,113 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import {
+  select,
+  scaleLinear,
+  axisBottom,
+  axisLeft,
+  line,
+  scaleTime,
+  format,
+  timeFormat,
+} from 'd3';
+
+import styles from './styles/historical-release-entries-line-plot.module.scss';
+
+// Specify the chart’s dimensions.
+const width = 400;
+const height = 400;
+const margin = { top: 20, right: 60, bottom: 45, left: 80 };
+
+export type DateCount = [Date, number];
+export type Bounds = {
+  date: [Date, Date];
+  count: [number, number];
+};
+
+type Props = {
+  dateCounts?: DateCount[];
+  bounds?: Bounds;
+};
+
+const HistoricalReleasesEntriesLinePlot = ({ dateCounts, bounds }: Props) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  const renderPlot = useCallback((dateCounts: DateCount[], bounds: Bounds) => {
+    const chart = select(svgRef.current).select('g');
```

Comment:
```suggestion
    const chart = select(svgRef.current).select('g');
    
    const startYear = new Date(bounds.date[0].getFullYear(), 0, 1);
    const endYear = new Date(bounds.date[1].getFullYear() + 1, 0, 1);
```

Modifying the bounds here and not in the original logic that is finding the data bounds because we're within the rendering component and this change of bounds is purely for visual effect